Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceUpdateExecutorInterface.h

```diff
@@ -30,6 +30,14 @@ namespace AzToolsFramework
 
             // Remove an Instance from the waiting queue.
             virtual void RemoveTemplateInstanceFromQueue(const Instance* instance) = 0;
+
+            // Set the executor as it never loads root prefab instance.
+            // A notification OnRootPrefabInstanceFirstLoadEnd will fire if root prefab instance
+            // is loaded for the first time after this function is called.
+            virtual void SetRootPrefabInstanceAsNeverLoaded() = 0;
```

Comment:
Hmm, I think something like `QueueRootPrefabLoadedNotificationForNextPropagation` might be a clearer indication of intention here, thoughts?

Reply:
Renamed.